web/src/settings_user_topics.ts

```diff
@@ -31,6 +32,8 @@ export function populate_list(): void {
         modifier_html(user_topic) {
             const context = {
                 user_topic,
+                topic_display_name: util.get_final_topic_display_name(user_topic.topic),
+                is_empty_topic_name: user_topic.topic === "",
```

Comment:
`is_empty_string_topic` might be a bit more readable than `is_empty_topic_name` for this templating variable.

Reply:
Renamed